src/prefect/agent/local/agent.py

```diff
@@ -83,7 +83,12 @@ def __init__(
             assert isinstance(self.labels, list)
             self.labels.append(hostname)
         self.labels.extend(
-            ["azure-flow-storage", "gcs-flow-storage", "s3-flow-storage"]
+            [
+                "azure-flow-storage",
+                "gcs-flow-storage",
+                "s3-flow-storage",
+                "github-flow-storage",
+            ]
```

Comment:
With the recent addition of env `image` metadata, these labels are starting to seem less desirable